### ISSUE TITLE
Outfit dialog sends hat=0-1

### ISSUE THREAD

**Issue by nhnb** [arianne/stendhal#377]
## 🐞 Describe the bug

According to the server log, the client sends a `outfit_ext` action with `hat=0-1`

## 🔥 How to reproduce

I was not able to reproduce the issue

## 🌍  Environment

- Web client

**Comment by AntumDeluge** [comment]
The problem is that the client is prepending "0" to outfit codes. "-1" is the value to use when a layer should not be drawn. So what is being sent is "0-1".
```
ERROR [verManager] CommandCenter            (165 ) Player [deluge, -1335443416] Attributes of Class(): [sourceid=68][zone=0_fado_forest_s_e2][zoneid=0_fado_forest_s_e2][type=outfit_ext][value=body=995,dress=01,head=0-1,mouth=00,eyes=0-1,mask=0-1,hair=0-1,hat=0-1]- Cannot execute action Attributes of Class(): [sourceid=68][zone=0_fado_forest_s_e2][zoneid=0_fado_forest_s_e2][type=outfit_ext][value=body=995,dress=01,head=0-1,mouth=00,eyes=0-1,mask=0-1,hair=0-1,hat=0-1] send by Player [deluge, -1335443416]
```

What should have been sent is:
```
value=body=995,dress=1,head=-1,mouth=0,eyes=-1,mask=-1,hair=-1,hat=-1
```

**Comment by AntumDeluge** [comment]
The problem appears to be here: https://github.com/arianne/stendhal/blob/88e9269/srcjs/stendhal/ui/outfitdialog.js#L482

This is unnecessary with the new outfit system.

**Comment by AntumDeluge** [comment]
I believe this is fixed by c9e12b0640, but should be tested more.

**Comment by AntumDeluge** [comment]
Working correctly on test server now. So closing this.
